src/OpenTelemetry/README.md

```diff
@@ -202,10 +202,24 @@ purposes, the SDK provides the following built-in processors:
 * [BatchExportProcessor&lt;T&gt;](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#batching-processor)
   : This is an exporting processor which batches the telemetry before sending to
   the configured exporter.
+
+  The following environment variables can be used to override the default
+  values of the `BatchExportActivityProcessorOptions`.
```

Comment:
might be a good idea to call out that this env variables affect *all* exporters unless individually overridden in code.